stock_prices/stock_prices.py

```diff
@@ -1,15 +1,29 @@
 #!/usr/bin/python
-
 import argparse
 
+
 def find_max_profit(prices):
-  pass
+    max_profit = 0
+    if len(prices) < 2:
+        return max_profit
+    for buy in range(len(prices)):
+        buy_price = prices[buy]
+        for sell in range((buy), len(prices)-1):
```

Comment:
I would try and see if there is a way you can turn this from a o(n^2) solution into an o(n) solution. Good first pass though :)

For me, I try to think of it as a bool flag setting to make me recognize if I am making or losing profit.